pkg/logical/expr_literal.go

```diff
@@ -52,12 +57,20 @@ func (i *int64Literal) String() string {
 	return strconv.FormatInt(i.int64, 10)
 }
 
-var _ Expr = (*int64ArrLiteral)(nil)
+var _ LiteralExpr = (*int64ArrLiteral)(nil)
 
 type int64ArrLiteral struct {
 	arr []int64
 }
 
+func (i *int64ArrLiteral) Bytes() [][]byte {
+	var b [][]byte
```

Comment:
how about `make` a slice with the capacity as `i.arr`

Reply:
I've created a slice with capacity, for both `int64ArrLiteral` and `strArrLiteral`.

Later, we may add a general byte buffer module to optimize memory allocation.